src/LinAlg/DiagMatrix.C

```diff
@@ -38,6 +38,17 @@ void DiagMatrix::dump (std::ostream& os, LinAlg::StorageFormat format, const cha
       utl::writeMatlab(label,myMat,os);
       break;
 
+    case LinAlg::MATRIX_MARKET:
+    {
```

Comment:
Not necessary with braces around this case block (since no local variable definitions within it). To be consistent with the other case-clocks and also for the similar added block for DenseMatrix above.

Reply:
removed.